qemu/tests/virtio_fs_localfs_migration.py

```diff
@@ -0,0 +1,169 @@
+import re
+
+from virttest import error_context, utils_disk, utils_misc, utils_test
+
+from provider import virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Basic migration test with diffirent cache modes over localfs
+    Steps:
+        1. Create shared directories on the host and write files
+        2. Run virtiofsd daemons on the host with different cache modes
+        3. Boot the source guest and mount the virtiofs targets in step1
+        4. Create different directories on the host and write files
+        5. Run virtiofsd daemons to share the directories in step4
+        6. Boot the target guest and mount the virtiofs targets in step4
```

Comment:
No need to mount the virtiofs targets here for the guest here should be paused for waiting migration.